Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:18
+
+RUN mkdir -p /home/node/app \
+    && chown -R node:node /home/node/app
+
+WORKDIR /home/node/app
+
+ENV NODE_ENV=production
+
+EXPOSE 3000
+
+USER node
+
+COPY --chown=node:node package.json /home/node/app/package.json
+COPY --chown=node:node yarn.lock /home/node/app/yarn.lock
+COPY --chown=node:node dist /home/node/app/dist
+
+RUN yarn install --production
```

Comment:
I'm not quite sure but since the env NODE_ENV is set to production, I don't think you need the `--production` argument (TBC)

Reply:
You're right https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install-production-true-false
Fixed